### PR TITLE
Update developer list

### PR DESCRIPTION
**Pull request by shinfan**
Reference: https://github.com/GoogleCloudPlatform/google-cloud-java/graphs/contributors

Please comment if I miss anyone.

Fixes https://github.com/GoogleCloudPlatform/google-cloud-java/issues/1503

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,36 +12,72 @@
...
       <organization>Google</organization>
       <roles>
         <role>Developer</role>
       </roles>
     </developer>
     <developer>
-      <id>mziccardi</id>
-      <name>Marco Ziccardi</name>
-      <email>mziccardi@google.com</email>
+      <id>michaelbausor</id>
```
Comment (vam-google) [inline pom.xml]: Marco and others are still responsible for the majority of the code written in this repo. I belive their names must stay here.

Comment (shinfan) [inline pom.xml]: According to the description:https://maven.apache.org/pom.html#Developers This developer list should be listing members who are **immediately responsible** for the code and if the person should not be contacted about the project, they need not be listed here.

**Comment by garrettjonesgoogle** [comment]
I think we should go with this. LGTM.

**Comment by mziccard** [comment]
@garrettjonesgoogle @shinfan @vam-google How could you remove me after all I did for you? :)

It's correct to remove us. You are doing a great job guys, keep it up!

**Comment by garrettjonesgoogle** [comment]
Hey @mziccard , nice to see you pop in :-)

**Pull request merged**